Translation/Extractor/TwigExtractor.php

```diff
@@ -165,6 +185,37 @@ public function getEntryComment($comments, $file, $line)
      */
     protected function extractFromDirectory($directory)
     {
-        return $this->getFinder()->files()->name('*.twig')->in($directory);
+        return $this->getFinder()->files()
+            ->name('*.twig')
+            ->in($directory)
+            ->exclude($this->getExcludedDirectories());
+    }
+
+    /**
+     * @param string|iterable $resource Files, a file or a directory
+     *
+     * @return iterable
+     */
+    protected function extractFiles($resource)
+    {
+        if (\is_array($resource) || $resource instanceof \Traversable) {
```

Comment:
You don't need the `\` before native functions
```suggestion
        if (is_array($resource) || $resource instanceof \Traversable) {
```

Reply:
Why did you change this line vs the one in the parent class?
```
if (is_iterable($resource)) {
```